public_html/wp-content/themes/pattern-directory/src/components/context-bar/index.js

```diff
@@ -73,18 +72,13 @@ function ContextBar( props ) {
 		}
 	}, [ query, isLoadingPatterns ] );
 
-	useEffect( () => {
-		const _height = message ? innerRef.current.offsetHeight : 0;
-		setHeight( _height );
-	}, [ message ] );
-
 	const classes = classnames( {
 		'context-bar__spinner': true,
 		'context-bar__spinner--is-hidden': ! isLoadingPatterns,
 	} );
 
-	return (
-		<header className="context-bar" style={ { height: `${ height }px` } }>
+	return ! message ? null : (
+		<header className="context-bar">
 			<div ref={ innerRef }>
```

Comment:
We'll want to get rid of this `ref`, it's no longer needed.

I think that `div` could also be removed and the styles applied to the header, but I'm not 100% on that.